Changes for RHDEVDOCS-4829 Doc: ODC Usability & Customer Happiness Epic (#55909)
openshift/openshift-docs: modules/odc-creating-helm-charts-using-developer-perspective.adoc

:_content-type: PROCEDURE

> [id="odc-creating-helm-charts-using-developer-perspective_{context}"]

Reviewer: ```suggestion
[id="odc-creating-helm-releases-using-developer-perspective_{context}"]
```

> = Creating Helm charts using the Developer perspective

Reviewer: ```suggestion
= Creating Helm releases using the Developer perspective
```

> You can use either the *Developer* perspective in the web console or the CLI to select and create a chart from the Helm charts listed in the *Developer Catalog*. You can create Helm releases by creating Helm charts and see them in the *Developer* perspective of the web console.

Reviewer: ```suggestion
You can use either the *Developer* perspective in the web console or the CLI to select and create a release from the Helm charts listed in the *Developer Catalog*. You can create Helm releases by installing Helm charts and see them in the *Developer* perspective of the web console.
```

Where available, you can switch between the *YAML View* and *Form View*. The data is persisted when switching between the views.
====
+

> .. Click *Create* to create a Helm release. You will be redirected to the *Topology* view where the release is displayed. If the Helm chart has release notes, the chart is pre-selected and the right panel displays the release notes for that release.

Reviewer: If a Helm Chart creates workloads when installed, then the user is redirected to Topology, else is navigated to the Helm release details page.
Workloads are : DaemonSet, CronJob, Pod, Deployment, DeploymentConfig

Author: @debsmita1 thank you for the review. I have updated all the suggestions by you. Just want to clarify whether this is an addition to the above step mentioned step. If yes, please let me know.
